### PR TITLE
Added Corey and Bethany as codeowners for repo.

### PR DESCRIPTION
**Pull request by BethanyG**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1,3 @@
-* @exercism/maintainers-admin
+* @cmccandless
+* @bethanyg
```
Comment (BethanyG) [inline .github/CODEOWNERS:2]:
```suggestion
* @cmccandless @BethanyG
```

**Comment by cmccandless** [comment]
I know we usually try to avoid specific usernames in CODEOWNERS files... maybe we need a `python-tooling` team that owns the test-runner, analyzer, and representer?

**Comment by BethanyG** [comment]
I think this is an intermediary thing while we are in development for V3, but will revert to the "main" codeowners long-term.  That being said, if you would prefer a group, we can do that.

**Comment by BethanyG** [comment]
@cmccandless  -- I see a team listed for `python-representer` team that includes myself, you, and @yawpitch.  Does that work, or do you want a new one?

**Comment by cmccandless** [comment]
I don't have a preference what the team is called; I'd say use the existing team. If it bothers @iHiD to have the team name scoped smaller than its access, a new one can be created.

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1 +1,2 @@
-* @exercism/maintainers-admin
+* @cmccandless @BethanyG
```
Comment (BethanyG) [inline .github/CODEOWNERS:1]:
```suggestion
* @exercism/python-representer
```

**Comment by iHiD** [comment]
I'd rather be explicit with names for now pls.

**Comment by iHiD** [comment]
(People I don't actively "trust" could rightfully get added to the `python-representer` team, but I would actively want to review their commits in tooling)

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1 +1,2 @@
-* @exercism/maintainers-admin
+* @exercism/python-representer
```
Comment (BethanyG) [inline .github/CODEOWNERS:1]:
```suggestion
* @cmccandless @BethanyG
```

**Comment by BethanyG** [comment]
ok.  reverted.

**Pull request merged**